AWSIoT/Internal/MQTTSDK/AWSMQTTDecoder.m

```diff
@@ -53,7 +53,7 @@ - (void)close {
 
 - (void)stream:(NSStream*)sender handleEvent:(NSStreamEvent)eventCode {
     AWSDDLogVerbose(@"%s [Line %d] EventCode:%lu, stream: %@, Thread: %@", __PRETTY_FUNCTION__, __LINE__, (unsigned long)eventCode, sender, [NSThread currentThread]);
-
+    int maxLengthMultiplier = 128 * 128 * 128;
```

Comment:
Not the right place to do this. It will still be computed every time there is an event. We should have it at a higher level that it is only computed once ideally.